libbpf-tools/Makefile

```diff
@@ -46,7 +47,6 @@ APPS = \
 	tcpconnect \
 	tcpconnlat \
 	vfsstat \
-	xfsslower \
```

Comment:
We still need to generate xfsslower. I think best way is to add something like

```
xfsslower ext4slower nfsslower btrfsslower: fsslower
    $(call msg,SYMLINK,$@)
    $(Q)ln -s $^ $@
```

Just make sure to also clean those up (probably would make sense to have `$(APP_ALIASES) = btrfsslower ext4slower ...` for that purpose.

Reply:
Updated.